frontend/src/components/endpoints/EndpointList.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import { Meta, Story } from '@storybook/react/types-6-0';
+import Endpoints, { KubeEndpoint } from '../../lib/k8s/endpoints';
+import { TestContext } from '../../test';
+import { generateK8sResourceList } from '../../test/mocker';
+import EndpointList from './List';
+
+Endpoints.useList = () => {
```

Comment:
Assigning it here would seem to modify the component, so if it's used elsewhere it will also use this stub function.

Reply:
I know. I will work on a way to make sure this happens only on mount and it's re-established to the original on unmount.
But for know should be fine because only the storybook runs this logic?